lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -521,6 +522,25 @@ def _section_cohort_management(course, access):
     return section_data
 
 
+def _section_discussions_management(course, access):
+    """ Provide data for the corresponding cohort management section """
+    course_key = course.id
+    ccx_enabled = hasattr(course_key, 'ccx')
+    section_data = {
+        'section_key': 'discussions_management',
+        'section_display_name': _('Discussions'),
+        'access': access,
```

Comment:
Out of curiosity, what is "access" here? I'm wondering if it actually needs to be passed through.